apps/ledger-live-desktop/src/renderer/components/CustomImage/CustomLockScreenDeviceAction/index.tsx

```diff
@@ -105,8 +105,10 @@ const CustomImageDeviceAction: React.FC<Props> = withRemountableWrapper(props =>
     if (error instanceof ImageCommitRefusedOnDevice) {
       dispatch(clearLastSeenCustomImage());
     }
-    onError && onError(error);
-  }, [dispatch, error, onError]);
+    if (!isRefusedOnStaxError) {
+      onError && onError(error);
+    }
+  }, [dispatch, error, onError, isRefusedOnStaxError]);
```

Comment:
[ASK] Why this change ? What happens now when the user refuses the picture on the device ?

Reply:
@ofreyssinet-ledger I simplified the logic now as the prev. condition was redundant